pkg/apiserver/util.go

```diff
@@ -19,32 +19,19 @@ package apiserver
 import (
 	"github.com/kubernetes-incubator/service-catalog/pkg/api"
 	servicecatalogrest "github.com/kubernetes-incubator/service-catalog/pkg/registry/servicecatalog/rest"
-	"github.com/kubernetes-incubator/service-catalog/pkg/registry/servicecatalog/server"
 	settingsrest "github.com/kubernetes-incubator/service-catalog/pkg/registry/settings/rest"
 	genericapiserver "k8s.io/apiserver/pkg/server"
 	"k8s.io/client-go/pkg/version"
-	restclient "k8s.io/client-go/rest"
 )
 
 const (
 	apiServerName = "service-catalog-apiserver"
 )
 
-func restStorageProviders(
-	defaultNamespace string,
-	storageType server.StorageType,
-	restClient restclient.Interface,
-) []RESTStorageProvider {
+func restStorageProviders() []RESTStorageProvider {
 	return []RESTStorageProvider{
-		servicecatalogrest.StorageProvider{
-			DefaultNamespace: defaultNamespace,
```

Comment:
In the future I strongly recommend splitting up your PRs based on related changes. This PR ideally should only have removed the storage type. And then in a separate PR remove unused things like `DefaultNamespace`. It's a lot more work for reviewers as the PR looks now to figure out what was a safe delete, vs. a refactor that tweaked a behavior unintentionally because a couple things are happening at the same time in this PR.

Reply:
The purpose of this PR was to simplify storage and align it with Kubernetes and other related projects. As part of the simplifying of storage in this PR, these fields became redundant and hence were removed. That seems to make sense to me as opposed to leaving redundant, unused fields that no longer relate to the refactored storage.